src/libs/actions/Session/index.js

```diff
@@ -316,7 +317,16 @@ function setPassword(password, validateCode, accountID) {
                 return;
             }
 
-            Onyx.merge(ONYXKEYS.ACCOUNT, {error: Localize.translateLocal('setPasswordPage.accountNotValidated')});
+            const login = lodashGet(response, 'data.email', null);
+            Onyx.merge(ONYXKEYS.ACCOUNT, {
+                validated: true, accountExists: true, validateCodeExpired: true, error: Localize.translateLocal('setPasswordPage.accountNotValidated'),
+            });
+
+            // This login is set with email received from the API so that the resend link can work if the user has hit a direct URL
```

Comment:
Maybe something like this instead:

>The login might not be set if the user hits a url in a new session. We set it here to ensure calls to resendValidationLink() will succeed.

Reply:
Done

Reply:
I see no changes 😄

Reply:
Was done didn't push. Hope you're able to see now.